src/pages/landing/index.tsx

```diff
@@ -65,6 +65,7 @@ const Landing = () => {
           className={styles.overlay}
           onClick={() => dispatch(handleMenu(false))}></div>
       )}
+      <Modal />
```

Comment:
Maybe is better to rename this component as ContactModal or something like that to be more specific and know what is for

Reply:
I agree, fixed!